### PR TITLE
Implementing 'use' verb

### PR DESCRIPTION
**Pull request by SaphiraSkies**
Not all item/feature inputs work yet, but the function processes input and errors now. First battery in flashlight works.

### CODE REVIEW

**Comment by SaphiraSkies** [comment]
Need to fix a bug, don't approve yet

**Comment by SaphiraSkies** [comment]
Nevermind. Haven't been able to replicate the error that I saw before. Might have happened before I merged the previous PRs into mine, so I'm leaving it be.

File: Language_Parser/actionVerbs.py
Comment (ABlair1) [inline Language_Parser/actionVerbs.py:454]: If we know the scenario represented by this `else:` condition will eventually fail and return `None` below, perhaps it would be better to just return `None` here (rather than handling the empty lists below)?

In other words, rather than assigning empty lists here (which will raise an exception below and return `None`), we could just return `None` in the `else:` condition. Please correct me if I'm misunderstanding something. Thanks!

**Review by ABlair1 — COMMENTED** [review]
Looks good. Please see comments for suggestions and let us know what you decide for each. Thanks!

File: Language_Parser/actionVerbs.py
Comment (SaphiraSkies) [reply Language_Parser/actionVerbs.py:385]: Later it will, yes. There are quite a few things I can't test until more of the conditionals are done. But I can make this a TODO to make sure it's not forgotten.

File: Language_Parser/actionVerbs.py
Comment (SaphiraSkies) [reply Language_Parser/actionVerbs.py:550]: Yes, that's correct.

File: Language_Parser/actionVerbs.py
Comment (SaphiraSkies) [reply Language_Parser/actionVerbs.py:357]: Haven't had a need for it at this time. The combineTwoItems function only needed it in order to pull new items from Game storage. Features don't have to do that, so I expect it to not be necessary.

File: Language_Parser/actionVerbs.py
Comment (SaphiraSkies) [reply Language_Parser/actionVerbs.py:530]: I chose the list instead because it specifically needs to hold an instance of Item, and as far as I'm aware, dicts only hold strings. It's a big part of why the function was separated into a helper function (to reduce complexity), so it also didn't make sense to translate strings to an Item instance on the receiving side.

I tried looking up how to have an instance of a class in a dict, but they just looked way more complicated than they needed to be for this purpose. So while I know it's not really fun to keep track of each index, it felt like the least awkward solution.

Any advice?

File: Language_Parser/actionVerbs.py
Comment (SaphiraSkies) [reply Language_Parser/actionVerbs.py:494]: Thanks!

File: Language_Parser/actionVerbs.py
Comment (SaphiraSkies) [reply Language_Parser/actionVerbs.py:454]: Yeah that's probably fair, I'll look into changing this.

File: Language_Parser/actionVerbs.py
Comment (ABlair1) [reply Language_Parser/actionVerbs.py:357]: Sounds good! I was referring to the docstring which says this function takes game data (as a parameter?)

File: Language_Parser/actionVerbs.py
Comment (ABlair1) [reply Language_Parser/actionVerbs.py:530]: Oh you can definitely store custom class objects as dictionary values. I'm pretty sure a Python dictionary can have any type object as a value. Here is an example I threw together really quickly:

image
image

Comment (ABlair1) [reply Language_Parser/actionVerbs.py:530]: This page has some more info about dictionaries in Python if you're curious. [https://realpython.com/python-dicts/#restrictions-on-dictionary-values](https://realpython.com/python-dicts/#restrictions-on-dictionary-values)

image

File: Language_Parser/actionVerbs.py
Comment (SaphiraSkies) [reply Language_Parser/actionVerbs.py:357]: Gotcha! I see what you mean, I'll fix it.

File: Language_Parser/actionVerbs.py
Comment (SaphiraSkies) [reply Language_Parser/actionVerbs.py]: You're right, it works. Not sure where I went wrong before, but it's fixed. Thanks!

**Comment by SaphiraSkies** [comment]
Should all be fixed.

**Review by ABlair1 — APPROVED** [review]

**Pull request merged**
